packages/ui/src/app/pages/Election/PastElections/PastElections.tsx

```diff
@@ -48,7 +48,7 @@ export const PastElections = () => {
             <>
               <PastElectionsListHeaders $colLayout={PastElectionsColLayout}>
                 <SortHeader {...getSortProps('cycleId')}>Round</SortHeader>
-                <SortHeader {...getSortProps('updatedAt')}>Election ended at</SortHeader>
+                <SortHeader {...getSortProps('updatedAt')}>Election started at</SortHeader>
```

Comment:
there is also sort property `startedAt` should we use it instead of `updatetAt`?  Otherwise  can we change the label to `Election updated at`?

Reply:
not sure about `startedAt` but `createdAt` is indeed the better choice:
`query { electionRounds { cycleId createdAt updatedAt endedAtTime } }`
```
    "electionRounds": [
      {
        "cycleId": 1,
        "createdAt": "2022-11-10T14:05:48.001Z",
        "updatedAt": "2022-11-25T15:28:00.001Z",
        "endedAtTime": "2022-11-25T15:28:00.001Z"
      },
      {
        "cycleId": 2,
        "createdAt": "2022-11-25T15:28:06.000Z",
        "updatedAt": "2022-11-25T15:28:06.000Z",
        "endedAtTime": null
      }
    ]
```

Reply:
must have misread the original task
> Fix the page to show election ended blockheight, and not started (as right now).